index.html

```diff
@@ -1,38 +1,76 @@
-<html>
+<!DOCTYPE html>
+<html lang="en">
 <head>
-<style>
-<link href="css/bootstrap.min.css" rel="stylesheet">
-body {font-family:Helvetica}
-</style>
+  <title>Buckeye Hackers</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+  <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>
 </head>
+
 <body>
-<div class="row">
-<div class="col-sm-6">
-    <a href="#"><img src="logo.png" alt="logo" height=300px></a>
-</div>
-<div class="col-sm-6">
-    <ul>
-        <li><a href="#about">About</a></li>
-        <li><a href="#calendar">Calendar</a></li>
-        <li><a href="#contact">Contact</a></li>
-    </ul>
-	</br>
-</div>
+  <div class="container-fluid vertical-center">
+    <div class="col-md-1"></div>
+
+<<<<<<< HEAD
```

Comment:
@tiranno as it appears you didn't realize you committed the `git diff` markers here and on lines 38 & 68